Tools-Override/crossgen.sh

```diff
@@ -2,7 +2,9 @@
 
 # Restores crossgen and runs it on all tools components.
 
-__CoreClrVersion=1.1.0
+__CoreClrVersion=2.0.0-preview1-25204-02
```

Comment:
The duplication of all these version is going to be a maintenance nightmare we need to think about ways of eliminating them.